packages/ses/src/error/types.js

```diff
@@ -315,8 +315,8 @@
  * @property {Console['table']} table
  * @property {Console['time']} time
  * @property {Console['timeEnd']} timeEnd
- * @property {Console['profile']} profile
- * @property {Console['profileEnd']} profileEnd
+ * property {Console['profile']} profile
```

Comment:
I had to comment these out or else Typescript from other projects would complain about their absence from the `Console` type.  Apparently they're not quite standard enough.

Reply:
Please add a comment in the code explaining this.